layouts/shortcodes/leadership.html

```diff
@@ -36,12 +40,49 @@ <h4>Ed Warnicke</h4>
     <img src="/img/headshots/dale.jpg" class="rounded-circle" title="Dale Davis Jones" />
       <h4>Dale Davis Jones</h4>
       <p>Vice President & Distinguished Engineer <br/> IBM</p>
+      <p>Community Workstream Lead</p>
   </div>
 
   <div class="mt-4 text-center col col-md-4 col-6">
-    <img src="/img/headshots/jason.png" class="rounded-circle" title="Jason Brooks" />
+    <img src="/img/headshots/jbrooks.png" class="rounded-circle" title="Jason Brooks" />
       <h4>Jason Brooks</h4>
       <p>Manager, Open Source Projects Office <br/> Red Hat</p>
+      <p>Community Workstream Lead</p>
+  </div>
+
+  <div class="mt-4 text-center col col-md-4 col-6">
+    <img src="/img/headshots/jstledger.png" class="rounded-circle" title="Jim St. Ledger" />
+      <h4>Jim St. Ledger</h4>
+      <p>Marketing Manager <br/> Intel</p>
+      <p>Marketing Workstream Lead</p>
+  </div>
+
+  <div class="mt-4 text-center col col-md-4 col-6">
+    <img src="/img/headshots/lkunz.png" class="rounded-circle" title="Larry Kunz" />
+      <h4>Larry Kunz</h4>
+      <p>Information Architect <br/> Extreme Networks</p>
+      <p>Language Workstream Lead</p>
+  </div>
+
+  <div class="mt-4 text-center col col-md-4 col-6">
+    <img src="/img/headshots/jlee.png" class="rounded-circle" title="Joanna Lee" />
+      <h4>Joanna Lee</h4>
+      <p>Partner <br/> Gesmer Upgrove LLP</p>
```

Comment:
```suggestion
      <p>Partner <br/> Gesmer Updegrove LLP</p>
```

Reply:
Corrected in latest force-push of commit.